Decisions of Polkadot Update (for JAM) (#1047)
Polkadot-Blockchain-Academy/pba-content: syllabus/7-Polkadot/1-decisions-of-polkadot-slides.md

<div>

Blockspace is the capacity of a blockchain to commit and finalize operations.

> An often overlooked problem is the economic scaling need to provide high quality blockspace.

Author: uhh, i guess the best point of elaboration is to read: 

https://www.rob.tech/blog/polkadot-blockspace-over-blockchains/

> Blockspace is not a commodity but rather a class of commodities. Blockspace produced by different systems will vary in quality, availability, and flexibility. The quality of blockspace can be judged by the security guarantees that the blockchain provides - the more secure, the higher the quality. Without a supply of blockspace, applications run into congestion or downtime, leading users to experience high fees, long wait times, or front-running. Without high-quality blockspace, applications are hacked and drained: low-quality blockspace is vulnerable to 51% attacks and toxic shock. Both types of occurrences will be familiar to readers who have spent time observing the blockchain ecosystem. These characteristics of blockspace are the key factors application developers must consider when choosing where to deploy.

My point in the slide is that shared security is what makes polkadot's blockspace "high quality" versus many other chains providing lots of blockspace too, but low quality.

> An often overlooked problem is the economic scaling need to provide high quality blockspace.

Reviewer: I think see your point but I feel like there's something wrong with the way this is worded in the slides. Economic scaling is a but undefined to me 

Based my understanding what you are saying is that blockspace quality is often sacrificed or diminished to gain more scalability, right?

Would you be okay with:
```suggestion
An often overlooked problem is that when pursuing scalability it is very easy to significantly diminish the quality of blockspace.
```

---

# JAM

---

## What is JAM?

JAM is the next evolution of the Polkadot protocol.

<img style="width: 600px;" src="../../assets/img/7-Polkadot/decisions/jam-pen-polkadot.png" />

The JAM chain will be the successor to the existing Polkadot Relay chain.

JAM will provide all the functionality of Polkadot v1, but also provide a more flexible foundation on which to build Web3 Applications.

---

## The Decisions of JAM

<div class="grid grid-cols-2">
<div>

**Unchanged**

- The Philosophies of Polkadot
- The Goals of Polkadot
  - Parallel Execution and Heterogenous Sharding
  - Shared Security
  - Interoperability

</div>

<div>

**Changed**

- Upgradability of the Core Protocol
  - Substrate -> JAM Chain

> - Parachains -> Services

Reviewer: Parachains are staying but there will be something new even more fundamental then them

> - Parachains -> Services

Author: yes, i had the same feeling when writing the line. let me think how to clarify

> - Parachains -> Services

Author: ```suggestion
- Services (as a superset of Parachains)
```

> - Parachains -> Services

Author: ```suggestion
- Services (as a superset of Parachains)
```

How about this?
